# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ooo/hotfix.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Currently not applying patches, ...

  - Rene Rebe <rene@exactcode.de>

--- ooo-build-2.1.13/patches/src680/apply.vanilla	2007-04-27 17:50:23.000000000 +0000
+++ ooo-build-2.1.13/patches/src680/apply	2007-04-27 17:53:48.000000000 +0000
@@ -119,10 +119,10 @@
 # Don't stat tons of config files we don't need to read on startup
 speed-configmgr.diff, i#56783, michael
 # Fix silly assertion bogosity
-cws-configmgrassert.diff, i#71663, michael
+#cws-configmgrassert.diff, i#71663, michael
 
 # accelerate all lookups
-speed-configmgr-isnamed.diff, i#73363, michael
+#speed-configmgr-isnamed.diff, i#73363, michael
 # accelerate 'getElementNames'
 speed-configmgr-getsimplerootname.diff, i#73364, michael
 
@@ -177,7 +177,7 @@
 dbaccess-default-varchar-lenght.diff, i#62664, jianhua
 
 #Improve the behavior of input field dialog.
-sw-input-field-dialog.diff, Zhang Yun
+#sw-input-field-dialog.diff, Zhang Yun
 
 # Improve the behavior of input fields inline editing.
 sw-input-field-inline-edit.diff, Zhang Yun
@@ -282,7 +282,7 @@
 
 # poke gnome-screensaver every 30 seconds during slideshow, n#169207, rodo
 vcl-gnome-screensaver-poke.diff, n#169207, rodo
-config_office-dbus-check.diff, n#169207, rodo
+#config_office-dbus-check.diff, n#169207, rodo
 
 # more helpful VCL debugging
 #vcl-debug.diff, i#72880, michael
@@ -821,7 +821,7 @@
 # localization if they are not translated
 sc-standard-filter-options-i18n-fake.diff, i#66473, pmladek
 
-recovery-disabled-crashreporter-localize.diff, i#65325, jholesov
+#recovery-disabled-crashreporter-localize.diff, i#65325, jholesov
 
 tango-icons-i18n.diff, n#254855, pmladek
 
@@ -1062,7 +1062,7 @@
 
 # allows to redefine the name of the UNIX wrapper that will be used
 # in the desktop files; needed for qstart
-config-unix-wrapper.diff, i#75366, n#252242, pmladek
+# config-unix-wrapper.diff, i#75366, n#252242, pmladek
 
 [ NotDebian ]
 # Improve readability of BUILD_VER_STRING in Help:About dialog
